guides/v2.1/config-guide/cli/config-cli-subcommands-i18n.md

```diff
@@ -99,22 +111,22 @@ The following table explains this command's parameters and values:
 |Parameter|Value|Required?|
 |--- |--- |--- |
 |`<source>`|Absolute file system path and file name of a .csv file that contains the combined translation dictionary and meta-information necessary for breakdown into a language package.<br><br>Use [`bin/magento i18n:collect-phrases`](#config-cli-subcommands-xlate-dict-dict) to create the .csv file then create the language package as discussed in [Create directories and files](#m2devgde-xlate-files).|Yes|
-|`<locale>`|[ISO 639-1](http://www.iso.org/iso/home/standards/language_codes.htm) (language) and [ISO 3166](http://www.iso.org/iso/country_codes.htm) (country) identifier of language used as file name for all resulting .csv files. Examples: `de_DE`, `pt_PT`, `pt_BR`.|Yes|
-|`-m --mode`|If a target file already exists, specifies whether to replace the existing language package or merge with the new language pack. Merging overrides any phrases that existed and adds new ones. <br><br>Values: merge or replace (the default).|No|
-|`-d --allow-duplicates`|Include this option to allow duplicates in the language pack. Otherwise, the command fails with an error if it encounters the same phrase is translated in different ways in different lines.|No|
+|`<locale>`|[ISO 639-1] (language) and [ISO 3166] (country) identifier of language used as file name for all resulting .csv files. Examples: `de_DE`, `pt_PT`, `pt_BR`.|Yes|
+|`-m --mode`|If a target file already exists, specifies whether to replace the existing language package or merge with the new language pack. Merging overrides any phrases that existed and adds new ones. <br><br>Values: merge or replace (default).|No|
+|`-d --allow-duplicates`|Include this option to allow duplicates in the language pack. Otherwise, the command fails with an error if it encounters the same phrase in multiple entries with different translations.|No|
 {:style="table-layout:auto;"}
 
 ### Create directories and files {#m2devgde-xlate-files}
 
-A language package is a directory under `app/i18n/<VendorName>` in the Magento file system with the following contents:
+Language packages are located in a directory under `app/i18n/<VendorName>` in the Magento file system with the following contents:
```

Comment:
It can be helpful to note the magento file system. There are settings, permissions, etc that go along with the file system and owner, doesn't hurt to be there.